main/manager.c

```diff
@@ -165,14 +165,32 @@
 			<parameter name="Username" required="true">
 				<para>Username to login with as specified in manager.conf.</para>
 			</parameter>
+			<parameter name="AuthType">
+				<para>Authorization type. Valid values are:</para>
+				<enumlist>
+					<enum name="plain"><para>Plain text secret. (default)</para></enum>
+					<enum name="MD5"><para>MD5 hashed secret.</para></enum>
+				</enumlist>
+			</parameter>
 			<parameter name="Secret">
-				<para>Secret to login with as specified in manager.conf.</para>
+				<para>Plain text secret to login with as specified in manager.conf.</para>
+			</parameter>
+			<parameter name="Key">
+				<para>Key to use with MD5 authentication.  To create the key, you must
+				first call the <literal>Challenge</literal> action, create an MD5 hash
+				with the result, then update the hash with the secret as specified
```

Comment:
What needs to be hashed specifically? The challenge response concatenated with the plain text password?

Reply:
Just as I described.  You need to create the hash first with the challenge response, then update the hash with with the secret.  Exactly how you do it depends on the technology you're using to interface with AMI.

If you're using python for instance...
```
import hashlib
m = hashlib.md5()
m.update(response_from_challenge)
m.update(your_secret)
key = m.hexdigest()
## '031edd7d41651593c5fe5c006fa5752b'
```

Do I need to include the example?

Reply:
Added python example to the description.